### PR TITLE
Css brochure form - Julie Dumbrique

### PR DESCRIPTION
**Pull request by juliedumbrique**
Screen Shot 2021-08-19 at 10 01 20 PM

### CODE REVIEW

File: css-brochure-form/index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  <div class="format">
+    <form>
+      <div>
+        <div>
+          <label for="email">COMPANY EMAIL</label>
+        </div>
+        <div>
+          <input type="text" name="email" id="email" placeholder="sally@company.co">
+        </div>
+      </div>
```
Comment (codingcodymiller) [inline css-brochure-form/index.html]: Whew, that is a lot of divs! This isn't necessarily a problem, but excessive use of `div` elements can make styling actually much more difficult. There's a joke we have in the developer community that overuse of divs is a disease known as `Divitis` 😂 

Try removing all but the necessary divs in your styling to try and keep your HTML more simple.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work on this!! One note for you:

**Comment by juliedumbrique** [comment]
Screen Shot 2021-08-20 at 3 51 08 PM

**Review by codingcodymiller — APPROVED** [review]
Much better!!! Excellent work! 🎉

**Pull request merged**